charts/riff/values.yaml

```diff
@@ -1,21 +1,28 @@
 ---
 cert-manager:
   enabled: true
-keda:
-  enabled: true
-knative:
-  enabled: true
+## Uncomment the following to disable the chart even if the tag is specified
+# keda:
+#   enabled: false
+## Uncomment the following to disable the chart even if the tag is specified
+# knative:
+#   enabled: false
```

Comment:
this file isn't user editable, so there really isn't a condition in which uncommenting makes sense

Reply:
it is of you do a `helm fetch` of the chart before installing, we could change the wording to include using `--set`